src/components/IpfsInput/IpfsInput.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { useMoralisFile } from 'react-moralis';
+import { IpfsInputprops } from './types';
+import { Upload } from '../Upload';
+
+const IpfsInput: React.FC<IpfsInputprops> = ({
```

Comment:
Could you change this component name to `IPFSInput`.
Sorry for being this picky 😅